Dockerfile

```diff
@@ -1,23 +1,38 @@
-FROM golang:1.20.14-alpine3.19
+FROM golang:1.20.14-alpine3.19 as build
 
-ARG GIT_COMMIT=unspecified
+ENV GOEXPERIMENT=arenas
+
+WORKDIR /opt/ceremonyclient
+
+COPY . .
+
+WORKDIR /opt/ceremonyclient/node
+
+RUN go install ./...
+RUN go install github.com/fullstorydev/grpcurl/cmd/grpcurl@latest
+
+FROM alpine:3.19
+
+ARG GIT_REPO
+ARG GIT_BRANCH
+ARG GIT_COMMIT
+
+ENV GOEXPERIMENT=arenas
 
 LABEL org.opencontainers.image.title="Quilibrium Network Node"
 LABEL org.opencontainers.image.description="Quilibrium is a decentralized alternative to platform as a service providers."
 LABEL org.opencontainers.image.vendor=Quilibrium
 LABEL org.opencontainers.image.url=https://quilibrium.com/
 LABEL org.opencontainers.image.documentation=https://quilibrium.com/docs
+LABEL org.opencontainers.image.source=$GIT_REPO
+LABEL org.opencontainers.image.ref.name=$GIT_BRANCH
 LABEL org.opencontainers.image.revision=$GIT_COMMIT
 
-ENV GOEXPERIMENT=arenas
-
-WORKDIR /opt/ceremonyclient
-
-COPY . . 
-
-WORKDIR /opt/ceremonyclient/node
+COPY --from=build /go/bin/node /usr/local/bin
+COPY --from=build /go/bin/grpcurl /usr/local/bin
+COPY --from=build /opt/ceremonyclient/node/ceremony.json /root
+COPY --from=build /opt/ceremonyclient/node/retroactive_peers.json /root
```

Comment:
@CassOnMars are there any other files that might be needed at runtime, other than the above two json files?

Reply:
No, there will be one last addition for 1.5, but I'll be sure to update this file with it